Packages/cdms2/Lib/dataset.py

```diff
@@ -245,14 +245,14 @@ def openDataset(uri,mode='r',template=None,dods=1,dpath=None, hostObj=None):
                 file = CdmsFile(uri,mode)
                 return file
             except:
-                raise "Error in DODS open of: ",uri
+                raise CDMSError, "Error in DODS open of: ",uri
         else:
             try:
                 datanode = loadURI(uri)
                 return datanode
             except:
                 datanode = loadURI(uri)
-                raise "Error in loadURI of: ",uri
+                raise CDMSError, "Error in loadURI of: ",uri
```

Comment:
`raise ExcType, argument` is kind of an old way to write things, but is compatible with 2.x versions; I sticked with it since this is what is used throughout this file.